src/common/no-deps.ts

```diff
@@ -12,12 +12,19 @@ export const parser = yargs(process.argv.slice(3))
 export interface BasicArguments extends YargsArguments {
   logLevel: string
   verbose: number
-  v: number
   skipCleanup: boolean
-  c: boolean
   trace: boolean
 }
 
+export const defaultBasicArguments: BasicArguments = {
+  _: [],
+  $0: 'defaultBasicArgs',
+  logLevel: 'WARN',
+  verbose: 0,
+  skipCleanup: false,
+  trace: false,
+}
+
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 export const asBool = (input: any): boolean => {
```

Comment:
why did you introduce this? Is `!!` not good enough? Please remove if this results in the same...